module2/homework/insensitivePalindrom/is_palindrome.hpp

```diff
@@ -0,0 +1,12 @@
+#include <algorithm>
+#include <cctype>
+#include <string>
+
+bool is_palindrome(std::string text)
+{
+    text.erase(std::remove_if(text.begin(), text.end(), [](unsigned char x) { return !std::isalpha(x); }), text.end());
```

Comment:
Do you even need a lambda here? Isn't isaplha a functor?

Reply:
+1